crates/lakekeeper/migrations/20250923164029_fuzzy_tabular_search.sql

```diff
@@ -0,0 +1,56 @@
+-- The purpose is enabling fuzzy search on `<namespace_name>.<tabular_name>`.
+--
+-- # Example use case
+--
+-- * User searches for `finance region 42`
+-- * Fuzzy search returns matches like
+--   * `departments.finance.region_42`
+--
+-- Column `namespace.namespace_name` is duplicated in tabular to enable a trigram index on
+-- `<namespace_name>.<tabular_name>`
+
+-- Add the new column (initially nullable)
+ALTER TABLE tabular
+ADD COLUMN namespace_name text[] COLLATE public.case_insensitive;
```

Comment:
We can't use `namespace_name` as a column, as this would then be ambiguous for namespace joins and thus break reads from older verions. This is why migration checks fail.
Switching to `tabular_namespace_name`